Updating GraphQL Mesh /reference (#75)
AdobeDocs/graphql-mesh-gateway: src/pages/reference/transforms/index.md

  Not all transforms require interception of both request and response. Some require straightforward mapping, so the runtime overhead could hopefully be negligible; however, there will always be some.

-  Multiple wrapping layers
  
  When using "wrap" mode, the required transformation can be achieved by adding at least one wrapping layer per each transform rule defined. We cannot have a wrapping layer per transform, but we need one per rule since each rule is unique in the way it transforms different parts of the schema. Some rules might even require multiple wrapping layers, f.i. When transforming a field, the transform needs to be applied to RootFields, ObjectFields, and InputObjectFields.

  As explained in the previous point, the wrapping layers are registered during initialization only. However, each wrapping layer will always have some runtime implications, even if hopefully negligible.

-  Working with fixed-schema sources

>     As mentioned, "wrap" is the only mode that works for sources that "speak" GraphQL natively. However, when you work with fixed schema sources, such as JSON-schema, OpenApi, SOAP, ..., schema wrapping might have some undesired effects; f.i. You won't have access to the original "fixed-contract" response from your data source.

Reviewer: ```suggestion
    As mentioned, "wrap" is the only mode that works for sources that "speak" GraphQL natively. However, when you work with fixed schema sources, such as JSON-schema, OpenApi, or SOAP. Schema wrapping might have some undesired effects. For example: you won't have access to the original "fixed-contract" response from your data source.
```
